Content.Server/Stack/StackSystem.cs

```diff
@@ -173,7 +173,7 @@ private void OnStackAlternativeInteract(EntityUid uid, StackComponent stack, Get
                 return;
             }
 
-            if (EntityManager.TryGetComponent<HandsComponent>(userUid, out var hands))
+            if (TryComp<HandsComponent>(userUid, out var hands))
             {
                 if (hands.TryGetActiveHeldEntity(out var heldItem) && heldItem != stack.Owner)
                 {
```

Comment:
Not directly related to the current PR, but IMO this whole hand check shouldn't be here, You should be able to take half of a stack sitting on a desk by alt clicking it.

And if the check stays, then it should be moved to the get-verbs logic, because currently it shows up in the verb menu even though it fails to do anything when the stack is not in your hands.

Reply:
Done, I simply made having hands optional. When you do have hands, it'll attempt to place the split stack in your hands or drop it. Otherwise, it just places it where you are.